requirements-py3.txt

```diff
@@ -3,6 +3,7 @@ lxml>=3.2.4
 pyOpenSSL>=0.13.1
 cssselect>=0.9
 queuelib>=1.1.1
+pathlib>=1.0.1
```

Comment:
In Python 3, pathlib is part of the standard library: https://docs.python.org/3/library/pathlib.html